storage/interceptable_repository.go

```diff
@@ -180,11 +180,26 @@ func (ir *queryScopedInterceptableRepository) GetForUpdate(ctx context.Context,
 }
 
 func (ir *queryScopedInterceptableRepository) List(ctx context.Context, objectType types.ObjectType, criteria ...query.Criterion) (types.ObjectList, error) {
-	objectList, err := ir.repositoryInTransaction.List(ctx, objectType, criteria...)
+	return ir.list(ctx, objectType, false, criteria...)
```

Comment:
very minor - maybe true would be better here in a sense that you say list with true meaning include labels and false meaning exclude